src/editor/tests.py

```diff
@@ -242,125 +242,247 @@ def test_editor_review_round_cancel(self):
         self.assertEqual(rounds.count(), 0)
 
     def test_editor_review_review_round(self):
-        book = core_models.Book.objects.get(pk=1)
-        self.assertEqual(book.stage.current_stage == 'review', True)
+        self.assertEqual(self.book.stage.current_stage, 'review')
+
         resp = self.client.get(
-            reverse('editor_review', kwargs={'submission_id': self.book.id}))
+            reverse(
+                'editor_review',
+                kwargs={
+                    'submission_id': self.book.id
+                }
+            )
+        )
         content = resp.content
         self.assertEqual(resp.status_code, 200)
         self.assertEqual("403" in content, False)
```

Comment:
I know this is not strictly part of the PR, but it might be nice to use `self.assertFalse("403" in content)` for this instead?

If you agree, there are a few other instances in the code where this happens.

Reply:
Yes, there are instances of this pattern throughout the existing tests in Rua. 

I initially changed a lot of them but James said that it wasn't the best use of time as the assertions still did what they were supposed to, however inelegantly. 

Eventually, I think these tests will need to be re-written or the long functions broken up into many much shorter functions with only one or maybe a few asserts in each. This might be a better time to correct everything.